gooey/_tmp/example_progress_bar_4.py

```diff
@@ -0,0 +1,39 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+
+from __future__ import unicode_literals
+from __future__ import print_function
+import sys
+from time import sleep
+from gooey import Gooey, GooeyParser
+
+
+@Gooey(progress_regex=r"^progress: (-?\d+)%$",
+       progress_animation=False)
```

Comment:
Hey, could you expand on this  `progress_animation`  a bit? I see that it's used in `base_window.py` with a comment about Windows 7, but I'm not totally clear on why it's in the decorator options.